projects/openshift.py

```diff
@@ -35,20 +37,37 @@
 @task(name='install-deps')
 def install_deps():
     install_packages([
-        'docker-io',
+        'docker',
+        'docker-registry',
         'git',
         'golang',
         'screen',
         'vim',
     ])
+    # required on CentOS7
+    sudo('yum -y update device-mapper')
 
 
 @task
-def deploy(proxy=''):
+def deploy():
     install_deps()
 
-    enable_services(['docker.socket'])
+    enable_services(['docker.service'])
+
     enable_tcp_ports([8443, 8444])
 
-    sudo('usermod -a -G docker {0}'.format(env.user))
+    sudo('groupadd docker')
+    sudo('usermod -aG docker {0}'.format(env.user))
+
     run('test -d origin || git clone https://github.com/openshift/origin.git')
+
+
+@task
+def start():
+    with cd('origin'):
+        run('make')
+        run('nohup ./_output/local/go/bin/openshift start &')
+
+        # add OpenShift as a provider in ManageIQ
+        create_provider('openshift01', env.host, {'user': 'admin',
+                                                  'password': 'smartvm'})
```

Comment:
Make these (admin/smartvm) defaults of `start` parameters. Even better if you move this to a different task.